examples/contextual_bandit/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+
+	wabbit "github.com/orktes/go-vowpal-wabbit"
+)
+
+func indexOf(val string, vals []string) int {
+	for i, v := range vals {
+		if val == v {
+			return i
+		}
+	}
+
+	return -1
+}
+
+func randomValue(vals []string) string {
+	return vals[rand.Intn(len(vals))]
+}
+
+func argMax(vals []float32) int {
+	maxIndx := 0
+	maxVal := float32(0)
+
+	for i, val := range vals {
+		if val > maxVal {
+			maxIndx = i
+		}
+	}
+	return maxIndx
+}
+
+func main() {
+	vw, err := wabbit.New("--cb_explore 4 --cover 3")
+	if err != nil {
+		panic(err)
+	}
+	defer vw.Finish()
+
+	contextValues := []string{"a", "b", "c", "d"}
+
+	wins := 0
+	tries := 0
+
+	for {
+		contextValue := randomValue(contextValues)
+
+		predictExample, err := vw.ReadExample(fmt.Sprintf(" | %s", contextValue))
+		if err != nil {
+			panic(err)
+		}
+
+		vw.Predict(predictExample)
+
+		scores := predictExample.GetActionScores()
+		selected := argMax(scores)
```

Comment:
I think this should not be argMax; it should be a sample from the returned distribution.